airbyte-cdk/python/airbyte_cdk/sources/streams/http/http.py

```diff
@@ -63,6 +63,27 @@ def http_method(self) -> str:
         """
         return "GET"
 
+    @property
+    def auto_fail_on_errors(self) -> bool:
```

Comment:
Probably "auto_raise_on_errors" or just "raise_on_errors" would be better to name it.

WDYT?

Reply:
changed to `raise_on_http_errors` . because retries still work on connection error, timeout error etc